client/TableSorting/Events.js

```diff
@@ -1,28 +1,29 @@
 //Events: click on Time button filter invoices
 Template.bTotal.events({
   'click button': function () {
-   SetSortParam("total");
+  	Session.set('invoiceSortBy', 'total');
+  	if (Session.get('invoiceSortOrder') == "asc")
+  	{
+  		Session.set('invoiceSortOrder','desc');
+  	}
+  	else if (Session.get('invoiceSortOrder') == "desc")
+  	{
+  		Session.set('invoiceSortOrder','asc');
+  	}
```

Comment:
you can shorten this `if` clause to:
```javascript
Session.set('invoiceSortOrder', Session.get('invoiceSortOrder') === "asc" ? "desc" : "asc");
```
please always use `===` instead of `==` as you learned in the Javascript test :wink: